getgauge/parser.py

```diff
@@ -0,0 +1,65 @@
+import os
+import sys
+import six
+from abc import ABCMeta, abstractmethod
+from .parser_parso import ParsoPythonFile
+from .parser_redbaron import RedbaronPythonFile
+
+
+class PythonFile(object):
+    Klass = None
```

Comment:
I'm not sure why the name is `Klass`. I think it can be `Class` since `Class` is not a keyword in python.